game/world/objects/npc.cpp

```diff
@@ -1097,8 +1098,8 @@ int32_t Npc::talentValue(Talent t) const {
   }
 
 int32_t Npc::hitChanse(Talent t) const {
-  if(t<Daedalus::GEngineClasses::MAX_HITCHANCE)
-    return hnpc.hitChance[t];
+  if(t<5)
```

Comment:
5 ? Original script has `MAX_HITCHANCE`, so should not be an issue to declare it as an enum

Reply:
Would it be fine with you to just replace it with `t <= Talent::TALENT_CROSSBOW`?